### ISSUE TITLE
Sources and timeline not always up to date

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#15]
**Reported by misterx on 16 Dec 2006 15:51 UTC**

Timeline and sources are sometimes not updated. At the moment, we are at revision 13 but timeline and sources only show revision 11.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 16 Dec 2006 15:51 UTC**

Error:

```
Repository checkins event provider (ChangesetModule) failed:

NoSuchChangeset: No changeset 10 in the repository
```

The changeset number changes and sometimes, a refresh of the page fixes the timeline (but still now showing the good revision)

**Comment by aircrack-ng** [comment]
**Comment by misterx on 16 Dec 2006 15:51 UTC**

This is only when also asking for repository checkins.
When this error appear (and changeset number changes) and browsing sources, revision numbers for some files changes.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 16 Dec 2006 15:51 UTC**

It should be fixed, I applied the patch from TRAC #4132 (but I had to apply some stuff manually).

Still waiting for the debian package to be sure that's really fixed (and maybe other things will be fixed in the package)

**Comment by aircrack-ng** [comment]
**Comment by gandalf on 16 Dec 2006 15:51 UTC**

Fixed too (trac upgraded to 0.10.3).
